docs/src/booklet/7-solvers.md

```diff
@@ -0,0 +1,29 @@
+# QUBO Solvers
+
+## Solvers, Annealers & Samplers
+[`ToQUBO.jl`](https://github.com/psrenergy/ToQUBO.jl)'s main goal is to make use of parameterized stochastic optimization solvers, particularly those relying on non-conventional hardware such as *Quantum Annealing* and other *Ising Machines*.
+A few `MOI`-compliant interfaces for annealers and samplers are bundled within [`ToQUBO.jl`](https://github.com/psrenergy/ToQUBO.jl) via the [`Anneal.jl`](https://github.com/psrenergy/Anneal.jl) companion package.
```

Comment:
This file still refers to Anneal.jl

Reply:
I replaced it on the docs, but not on tests. We have to finish writing QUBODrivers before propagating changes to the other ones.